src/routes/data/index.svelte

```diff
@@ -68,6 +68,21 @@
   </div>
 </div>
 
+<div class="bx--grid">
+  <div class="bx--row margin--v-32">
+    <div class="bx--col-lg-2"></div>
+    <div class="bx--col-lg-9">
+      <!-- Fifth Assessment Notification -->
+      <AlertLink
```

Comment:
Instead of passing the alert content as props, another option is to create a specifc Fifth Assessment Data alert partial with hard coded values. That way you don't have to update the language in the different places where this alert is used. 

But the way you have it fine too. You just need to make sure the prop values are updated in all the pages.

Reply:
@mukhtyar I created it this way so we can have a new reusable component for other purposes in the future. I know it would have been helpful to make changes in this text, but I just did a find-and-replace in this case.